gothemis/compare/compare.go

```diff
@@ -77,6 +77,13 @@ const (
 
 // Errors returned by Secure Comparator.
 var (
+	ErrAppendSecret  = errors.New("Failed to append secret")
+	ErrCreateObj     = errors.New("Failed to create comparator object")
+	ErrDestroyObj    = errors.New("Failed to destroy comparator object")
```

Comment:
in most cases we try to name variables and errors with their full name and try to keep such way everywhere. Or we should name using shortnames, or we should use fullnames. But not both because it add new problem "how I should name this variable, with short or full name?". We don't pay for every character which we use in code)

Reply:
No problem, gonna update s/Obj/Object/g, s/Rem/Remote/g, etc. Wasn't sure whether long names were OK